src/Linter/ProcessLinterProcessBuilder.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Linter;
+
+use Symfony\Component\Process\Process;
+use Symfony\Component\Process\ProcessBuilder;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @internal
+ */
+final class ProcessLinterProcessBuilder
+{
+    /**
+     * @var string
+     */
+    private $executable;
+
+    /**
+     * @param string $executable PHP executable, null for autodetection
+     */
+    public function __construct($executable)
+    {
+        $this->executable = $executable;
+    }
+
+    /**
+     * @param string $path
+     *
+     * @return Process
+     */
+    public function build($path)
+    {
+        // @TODO change at 2.4 to:
```

Comment:
The point is to have those listed by IDEs (which are pretty lenient with this) and ultimately removed, so I'd say this is perfectly fine.